### DISCUSSION TITLE
03/038-%E7%AC%AC%E4%B8%89%E5%8D%81%E5%85%AB%E7%AB%A0-%E6%8A%8A%E6%8F%A1%E4%BB%8A%E5%A4%A9

### DISCUSSION THREAD

**Discussion by giscus** [liuli-moe/to-the-stars#65] [General]
# 03/038-%E7%AC%AC%E4%B8%89%E5%8D%81%E5%85%AB%E7%AB%A0-%E6%8A%8A%E6%8F%A1%E4%BB%8A%E5%A4%A9

在经历了几个世纪的动荡之后，一个乌托邦式的 AI— 人类政府治理着地球，预示着后稀缺社会的来临和太空殖民的新时代。一次意外的接触却让科技更先进的敌对外星种族打破了和平，这迫使魔法少女们走出幕后，拯救人类文明。在这一切之中，志筑良子，一个普通的女孩，仰望着星空，好奇着她在宇宙中的归所。

https://tts.liuli.moe/03/038-%E7%AC%AC%E4%B8%89%E5%8D%81%E5%85%AB%E7%AB%A0-%E6%8A%8A%E6%8F%A1%E4%BB%8A%E5%A4%A9.html

**Comment by 5424274** [comment]
“像现在这样的种种瞬间，探索着一个又一个的星球，接吻，还有你懂的，那个。一切的一切。我感觉完整的一生就摆在我们眼前，而我想要充分品味我们所能拥有的全部。永恒的时间就在我们手中，良子 —— 就像执政体所说，曾经有亿万的祖先埋葬于九千之下，没能享受到我们这样的好机会。但现在我却感到，我们跑来做这种任务简直就像是轻易地抛弃了所有的这一切。”

九千之下，应该是九泉之下吧。
